src/kagglehub/clients.py

```diff
@@ -93,18 +93,18 @@ def _download_file(response: requests.Response, out_file: str, size_read: int, t
 
 
 # These environment variables are set by the Kaggle notebook environment.
-KAGGLE_URL_BASE_ENV_VAR_NAME = "KAGGLE_URL_BASE"
+KAGGLE_DATA_PROXY_URL_ENV_VAR_NAME = "KAGGLE_DATA_PROXY_URL"
```

Comment:
FYI, only the `AttachDatasourceUsingJwtHandler` is whitelisted to be called through the data proxy: https://github.com/Kaggle/kaggleazure/blob/7a9792daf86265c9a78732af963c0c3331f8920c/Kaggle.Emulators.DataProxy/Controllers/KaggleJwtHandlerController.cs#L33